index.js

```diff
@@ -29,4 +30,57 @@ if (whitelist.length > 0 || blacklist.length > 0) {
 const customPlatformsConfig = require('./custom_platforms.js');
 const platformsPath = path.join(__dirname, './platforms');
 
-compiler.compile(filtersDir, logPath, reportPath, platformsPath, whitelist, blacklist, customPlatformsConfig);
+/**
+ * Search for specific files by extension type and
+ * replaces all occurrencies of necessary strings to new ones in the build results.
+ */
+
+ async function replaceInFilters() {
+/**
+ *  
+ * @param {String} dir - specific directory that limits the search.
```

Comment:
`string` should be lower-case 

`searchFiles()` should also have a description

Reply:
Done.